tests/src/performance/Scenario/JitBench/JitBenchHarness.cs

```diff
@@ -0,0 +1,485 @@
+            return new Dictionary<string, string> { { "PATH", $"{Path.Combine(s_jitBenchDevDirectory, ".dotnet")};{psi.Environment["PATH"]}" } };
+        }
+
+        private static void InstallDotnet()
+        {
+            var psi = new ProcessStartInfo() {
+                WorkingDirectory = s_jitBenchDevDirectory,
+                FileName = @"powershell.exe",
+                Arguments = @".\Dotnet-Install.ps1 -InstallDir .dotnet -Channel master -Architecture x64"
+            };
+            LaunchProcess(psi, 180000);
+        }
+
+        private static void ModifySharedFramework()
+        {
+            // Current working directory is the <coreclr repo root>/sandbox directory.
+            Console.WriteLine($"Modifying the shared framework.");
+
+            var sourcedi = new DirectoryInfo(Directory.GetCurrentDirectory());
+            var targetdi = new DirectoryInfo(
+                new DirectoryInfo(Path.Combine(s_jitBenchDevDirectory, ".dotnet", "shared", "Microsoft.NETCore.App"))
+                .GetDirectories("2.1*").First().FullName);
+
+            Console.WriteLine($"  Source : {sourcedi.FullName}");
+            Console.WriteLine($"  Target : {targetdi.FullName}");
+
+            var compiledBinariesOfInterest = new string[]
+            {
+                "clretwrc.dll",
+                "clrjit.dll",
+                "coreclr.dll",
+                "mscordaccore.dll",
+                "mscordbi.dll",
+                "mscorrc.debug.dll",
+                "mscorrc.dll",
+                "sos.dll",
+                "SOS.NETCore.dll",
+                "System.Private.CoreLib.dll"
+            };
+
+            foreach (var compiledBinaryOfInterest in compiledBinariesOfInterest)
+            {
+                foreach (FileInfo fi in targetdi.GetFiles(compiledBinaryOfInterest))
+                {
+                    var sourceFilePath = Path.Combine(sourcedi.FullName, fi.Name);
+                    var targetFilePath = Path.Combine(targetdi.FullName, fi.Name);
+
+                    if (File.Exists(sourceFilePath))
+                    {
+                        File.Copy(sourceFilePath, targetFilePath, true);
+                        Console.WriteLine($"    Copied file - '{targetFilePath}'");
+                    }
+                }
+            }
+        }
+
+        private static IDictionary<string, string> GenerateStore(IDictionary<string, string> environment)
+        {
+            // This step generates some environment variables needed later.
+            var environmentFileName = "JitBenchEnvironment.txt";
+            var psi = new ProcessStartInfo() {
+                WorkingDirectory = s_jitBenchDevDirectory,
+                FileName = "powershell.exe",
+                Arguments = $"-Command \".\\AspNet-GenerateStore.ps1 -InstallDir .store -Architecture x64 -Runtime win7-x64; gi env:JITBENCH_*, env:DOTNET_SHARED_STORE | %{{ \\\"$($_.Name)=$($_.Value)\\\" }} 1>>{environmentFileName}\""
```

Comment:
Currently, there is no way to get this back from the Api/Process. This is a workaround. Maybe we should open an issue.